scripts/lint-python

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Run pylint on all python code
+
+# Exit on error
+set -e
+
+# Exit on pipe fail
+set -o pipefail
+
+PYLINT=${PYLINT:-pylint}
```

Comment:
What's the overlap with `.ci/check-pep8`? that script should probably use this script, right?

maybe even use the other script as a base, so mypy etc is also checked, but then have the .ci/ script just call it with the diff files (or on all of them to keep it simple, as linting Python does not take any time).

Reply:
I guess it wouldn't hurt to have this script take a list of files as optional input.